recipes/icalendar-searcher/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "icalendar-searcher" %}
+{% set version = "1.0.3" %}
+{% set python_min = "3.9" %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default.

```suggestion
```